compatibility/versions.bzl

```diff
@@ -27,7 +27,7 @@ sdk_versions = [
     "2.2.1",
     "2.3.8",
     "2.4.2",
-    "2.5.1",
```

Comment:
this file is autogenerated from `UpdateVersion.hs`. I think if you want to catch this, then update the script to include both 2.5.0 and 2.5.1 in there.

Reply:
Good catch.  I was not aware of this Haskell file.